src/js/mapper/CAELiveMapper.js

```diff
@@ -85,36 +91,49 @@ function CAELiveMapper() {
                 }).show();
             }
 
-            if (!widgetNodeId) {
-                widgetNodeId = SyncMeta.createNode('Widget', 4500, 4500, 100, 100, 1);
-                setTimeout(function () {
-                    var meta = editor.graph.model.getMeta();
-                    SyncMeta.setAttributeValue(widgetNodeId, 'width', Util.formatNumber(meta.getAttribute('width')));
-                    SyncMeta.setAttributeValue(widgetNodeId, 'height', Util.formatNumber(meta.getAttribute('height')));
-                }, 750);
-            } else {
-                var meta = editor.graph.model.getMeta();
-                /*for (var i = 0; i < meta.attributes.length; i++) {
-                    var attrib = meta.attributes[i];
-                    var ytext = widgetYMap.get(widgetNodeId + '[' + attrib.name.slice(1) + ']');
-                    if(ytext)
-                        widgetAttr[attrib.name] = ytext;
-                    else mxLog.writeln('Widget node attribute ' + attrib.name + ' is not ytext????');
-                }*/
-                SyncMeta.setAttributeValue(widgetNodeId, 'width', Util.formatNumber(meta.getAttribute('width')));
-                SyncMeta.setAttributeValue(widgetNodeId, 'height', Util.formatNumber(meta.getAttribute('height')));
-            }
-
-            editor.graph.model.setAttribute('id', widgetNodeId);
+            // if (!widgetNodeId) {
```

Comment:
Can these commented out lines be removed?

Reply:
yes

Reply:
resolved